hadoop-hdds/framework/src/main/java/org/apache/hadoop/ozone/common/BlockGroup.java

```diff
@@ -88,13 +81,13 @@ public static BlockGroup getFromProto(KeyBlocks proto) {
   }
 
   public static BlockGroup getFromBlockIDProto(KeyBlocks proto) {
-    List<BlockID> blockIDs = new ArrayList<>();
+    List<DeletedBlock> deletedBlocks = new ArrayList<>();
```

Comment:
We can combine getFromDeletedBlockProto() as another loop here only, as will get either getBlocksList() or getDeletedBlocksList() from OM.